qutebrowser/utils/utils.py

```diff
@@ -856,3 +856,18 @@ def expand_windows_drive(path):
         return path + "\\"
     else:
         return path
+
+def is_mac():
+    if sys.platform == 'darwin':
+        return True
+    return False
+
+def is_linux():
```

Comment:
Per https://docs.python.org/2/library/sys.html#sys.platform, I think it might be better to use `startswith` for linux/mac as well.

Reply:
I've added ```startswith()``` for linux and mac as well.